source/_integrations/home_connect.markdown

```diff
@@ -103,7 +103,50 @@ The integration configuration will ask for the *Client ID* and *Client Secret* c
+| `program` | yes | Program to select. If set, it will start or select a program depending on `affects_to`. |
+| `consumer_products_cleaning_robot_option_reference_map_id` | yes | Defines the used reference map. |
+| `consumer_products_cleaning_robot_option_cleaning_mode` | yes | Defines the favoured cleaning mode. |
+| `consumer_products_coffee_maker_option_bean_amount` | yes | Describes the bean amount of a coffee machine program. |
+| `consumer_products_coffee_maker_option_fill_quantity` | yes | Describes the fill quantity (in ml) of a coffee machine program. |
+| `consumer_products_coffee_maker_option_coffee_temperature` | yes | Describes the coffee temperature of a coffee machine program. |
+| `consumer_products_coffee_maker_option_bean_container` | yes | Defines the preferred bean container. |
+| `consumer_products_coffee_maker_option_flow_rate` | yes | Defines the water-coffee contact time. The duration extends to coffee intensity. |
+| `consumer_products_coffee_maker_option_multiple_beverages` | yes | Defines if double dispensing is enabled. |
+| `consumer_products_coffee_maker_option_coffee_milk_ratio` | yes | Defines the milk amount. |
+| `consumer_products_coffee_maker_option_hot_water_temperature` | yes | Defines the temperature suitable for the type of tea. |
+| `b_s_h_common_option_start_in_relative` | yes | Defines in how many time the program should start. |
+| `dishcare_dishwasher_option_intensiv_zone` | yes | Defines if the cleaning is done with higher spray pressure on the lower basket for very dirty pots and pans. |
+| `dishcare_dishwasher_option_brilliance_dry` | yes | Defines if the program sequence is optimized with special drying cycle ensures more shine on glasses and plastic items. |
+| `dishcare_dishwasher_option_vario_speed_plus` | yes | Defines if the program run time is reduced by up to 66% with the usual optimum cleaning and drying. |
+| `dishcare_dishwasher_option_silence_on_demand` | yes | Defines if the extra silent mode is activated for a selected period of time. |
+| `dishcare_dishwasher_option_half_load` | yes | Defines if economical cleaning is enabled for smaller loads which reduces energy and water consumption and also saves time. The utensils can be placed in the upper and lower baskets. |
+| `dishcare_dishwasher_option_extra_dry` | yes | Defines if improved drying for glasses and plasticware is enabled. |
+| `dishcare_dishwasher_option_hygiene_plus` | yes | Defines if the cleaning is done with increased temperatures which ensures maximum hygienic cleanliness for regular use. |
+| `dishcare_dishwasher_option_eco_dry` | yes | Defines if the door is opened automatically for extra energy efficient and effective drying. |
+| `dishcare_dishwasher_option_zeolite_dry` | yes | Defines if the program sequence is optimized with special drying cycle ensures improved drying for glasses, plates and plasticware. |
+| `laundry_care_dryer_option_drying_target` | yes | Describes the drying target for a dryer program. |
```

Comment:
what is a drying target? are there options, like moist, dry, extra dry?

Reply:
from https://api-docs.home-connect.com/programs-and-options/#dryer_drying-target-option, the possible options are:

- Iron Dry
- Gentle Dry
- Cupboard Dry
- Cupboard Dry Plus
- Extra Dry